triggers/service/auth/src/test/scala/com/daml/auth/middleware/oauth2/TestMiddleware.scala

```diff
@@ -43,6 +46,13 @@ abstract class TestMiddleware
     with SuiteResourceManagementAroundAll
     with Matchers
     with OptionValues {
+
+  val authenticationSecurity: SecurityTest =
+    SecurityTest(property = Authentication, asset = "TBD")
+
+  val authorizationSecurity: SecurityTest =
+    SecurityTest(property = Authorization, asset = "TBD")
+
```

Comment:
Can you link an issue or at least add a comment about this TDB. I have no idea what this is about.